src/main/resources/static/js/scripts.js

```diff
@@ -8,4 +8,11 @@ function toggleSection(val, contextPath) {
     $("#rosterLink").attr("href", contextPath + "/roster/" + val);
     $("#classSetupLink").attr("href", contextPath + "/courseConfiguration/" + val);
     $("#sectionSelect").attr("action", contextPath + "/roster/" + val + "/save");
+}
+
+function toggleButton(){
```

Comment:
maybe update the name of this function to describe what it does, like disablePushConfirmation. Just a minor readability improvement.